lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -46,10 +46,10 @@ def initialize(term_config)
       def find(id, request_header: {}, language: nil, replacements: {}, subauth: nil, format: 'json', performance_data: false) # rubocop:disable Metrics/ParameterLists
         request_header = build_request_header(language: language, replacements: replacements, subauth: subauth, format: format, performance_data: performance_data) if request_header.empty?
         unpack_request_header(request_header)
-        raise Qa::InvalidLinkedDataAuthority, "#{request_id} - Unable to initialize linked data term sub-authority #{subauthority}" unless subauthority.nil? || term_subauthority?(subauthority)
+        raise Qa::InvalidLinkedDataAuthority, "Unable to initialize linked data term sub-authority #{subauthority}" unless subauthority.nil? || term_subauthority?(subauthority)
         @id = id
         url = authority_service.build_url(action_config: term_config, action: :term, action_request: normalize_id, request_header: request_header)
-        Rails.logger.info "#{request_id} - QA Linked Data term url: #{url}"
+        Rails.logger.info "QA Linked Data term url: #{url}"
```

Comment:
There are several places where `#{request_id}` is removed.  The same effect can be achieved with standard logging features.  See PR notes for more info.